sovtokenfees/sovtokenfees/test/catchup/revert/test_revert_fees.py

```diff
@@ -148,18 +155,28 @@ def test_revert_set_fees_and_view_change_all_nodes(nodeSetWithIntegratedTokenPlu
     nodes = nodeSetWithIntegratedTokenPlugin
     node_set = [n.nodeIbStasher for n in nodeSetWithIntegratedTokenPlugin]
     seq_no = get_seq_no(xfer_mint_tokens)
-    _old_pp_seq_no = get_ppseqno_from_all_nodes(nodeSetWithIntegratedTokenPlugin)
+    _old_len_pprs = set([get_len_preprepares(n) for n in nodeSetWithIntegratedTokenPlugin])
+    assert len(_old_len_pprs)
+    _old_len_ppr = _old_len_pprs.pop()
 
-    with delay_rules_without_processing(node_set, cDelay()):
+    with delay_rules_without_processing(node_set, pDelay(), cDelay()):
         helpers.general.set_fees_without_waiting({NYM: 5})
-        looper.run(eventually(functools.partial(check_batch_ordered, _old_pp_seq_no, nodeSetWithIntegratedTokenPlugin)))
-        ensure_view_change(looper, nodes)
-
-    ensureElectionsDone(looper=looper, nodes=nodes)
+        looper.run(eventually(functools.partial(check_batch_ordered, _old_len_ppr, nodeSetWithIntegratedTokenPlugin)))
+        send_and_check_nym_with_fees(helpers,
+                                     {FEES: {NYM: 5}},
+                                     seq_no,
+                                     looper,
+                                     xfer_addresses,
+                                     1000,
+                                     check_reply=False)
+        for n in nodeSetWithIntegratedTokenPlugin:
```

Comment:
Maybe it's better to do just a catch-up, not a view change? Then we check that revert works correctly and old FEEs are restored without manual revert of requests and batches.